README.md

```diff
@@ -119,6 +119,86 @@ public class UserNotificationGrain : Grain<UserNotificationState>, IUserNotifica
   }
 }
 ```
+# Example configuration cohosting aspnetcore website and orleans
+
+```cs
+using System.Linq;
+using FFT.SignalServer;
```

Comment:
Can we remove that extra package from the sample? Just for the sake of keep it clean. Thanks!

Reply:
Yes of course! I'll try and figure out how to edit it now :)